diff --git a/arch/arm/boot/dts/vexpress-v2p-ca9.dts b/arch/arm/boot/dts/vexpress-v2p-ca9.dts
index c9305b58..b7bf8d7f 100644
--- a/arch/arm/boot/dts/vexpress-v2p-ca9.dts
+++ b/arch/arm/boot/dts/vexpress-v2p-ca9.dts
@@ -29,6 +29,23 @@
 		i2c1 = &v2m_i2c_pcie;
 	};
 
+	chrdev {
+		compatible = "ldddc,chrdev";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		chrdev@2 {
+			label = "cdev-eeprom";
+			reg = <2>;
+		};
+
+		chrdev@4 {
+			label = "cdev-rom";
+			reg = <4>;
+			read-only;
+		};
+	};
+
 	cpus {
 		#address-cells = <1>;
 		#size-cells = <0>;
